src/config/dark-sites.config

```diff
@@ -402,6 +402,7 @@ gifyourgame.com
 giggl.app
 gikken.co
 giphy.com
+github.dev
```

Comment:
Not dark-by-default.

![image](https://user-images.githubusercontent.com/25481501/177014409-5581a1fa-c8e3-4d23-9fda-ab00f8043fd7.png)

Reply:
Maybe it respects what you selected on the main GitHub domain, did you selected the GitHub to use the dark theme?

Reply:
Nope light-mode. But that isn't dark-by-default.

Reply:
Honestly, I think if you're going to open the a Codespace, you already have a GitHub account and (probably) put the main website on the Dark mode too.
But, that's my personal opinion, and I can't speak for everyone.

Anyways, I already pushed the new commit fixing it